### PR TITLE
added code & tests

### PR DESCRIPTION
**Pull request by SianCulligan**

### CODE REVIEW

**Comment by SianCulligan** [comment]
- [x] Top-level README “Table of Contents” is updated
 - [x] Feature tasks for this challenge are completed
 - [x] Unit tests written and passing
     - [x] “Happy Path” - Expected outcome
 - [x] README for this challenge is complete
     - [x] Summary, Description, Approach & Efficiency, Solution
     - [x] Link to code
     - [x] Picture of whiteboard (BLOG)

File: _tests_/mergeSort.test.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+'use strict';
+
+const mergeSort = require('./mergeSort.js');
+
+describe(mergeSort, () => {
+    it('A function that sorts an array from the middle.', () => {
+    let str = insertionSort([8,4,23,16,15]);
```
Comment (lizkavalski) [inline _tests_/mergeSort.test.js:7]: I think you mean mergeSort instead of insertionSort.

File: mergeSort.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
+function mergeSort(arr, sIndx, eIndx) {
```
Comment (lizkavalski) [inline mergeSort.js:1]: should just take in the unsorted array.

File: mergeSort.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+  let merged = [];
+  let mid = Math.floor(eIndx - sIndx / 2);
+  let j = mid + 1;
```
Comment (lizkavalski) [inline mergeSort.js:4]: The first part of the function should  define the left ,right and middle of the array. then use the merge function that take in the left,right and the array. The merge function is where comparing of the values is happening.

**Pull request merged**